qa-test-apps/helm-app/values-files-with-no-values/values.yaml

```diff
@@ -0,0 +1,2 @@
+key: key1
```

Comment:
Is this a leftover? Or deliberately left here to demonstrate that it is not used? :thinking:

Reply:
Yes, it was kept as default values. Later those are changed by either `valuesFiles` or `valuesFrom`.